mitgliedsantrag.html

```diff
@@ -54,12 +54,12 @@ <h2>Mitgliedsantrag</h2>
             </div>
             <hr />
             <img style="display: unset;"
-              src="images/mitgliedsantragvorschau.PNG"
+              src="images/mitgliedsantrag_vorschau.JPG"
```

Comment:
Sieht aktuell so aus:
![image](https://github.com/jheinath/TTVB-Homepage/assets/44709506/ed09f010-6312-44d0-b1dc-c2cc98f5fa56)
Das Vorschaubild liegt nämlich im Ordner "other" und nicht in "images"

Reply:
Vorschaubild nach "images" verschoben